packages/api-client/src/views/Cookies/Cookies.vue

```diff
@@ -5,44 +5,67 @@ import SidebarList from '@/components/Sidebar/SidebarList.vue'
 import SidebarListElement from '@/components/Sidebar/SidebarListElement.vue'
 import ViewLayout from '@/components/ViewLayout/ViewLayout.vue'
 import ViewLayoutContent from '@/components/ViewLayout/ViewLayoutContent.vue'
-import { useSidebar } from '@/hooks'
 import type { HotKeyEvent } from '@/libs'
-import { useWorkspace } from '@/store'
-import { ScalarIcon } from '@scalar/components'
+import { useActiveEntities, useWorkspace } from '@/store'
+import { useModal } from '@scalar/components'
 import { type Cookie, cookieSchema } from '@scalar/oas-utils/entities/cookie'
 import { computed, onBeforeUnmount, onMounted } from 'vue'
 import { useRoute, useRouter } from 'vue-router'
 
 import CookieForm from './CookieForm.vue'
-import CookieRaw from './CookieRaw.vue'
+import CookieModal from './CookieModal.vue'
 
-const { cookies, cookieMutators, events } = useWorkspace()
-const { collapsedSidebarFolders, toggleSidebarFolder } = useSidebar()
+// import CookieRaw from './CookieRaw.vue'
```

Comment:
```suggestion
```
old import?

Reply:
We might want to bring it back soon, just felt too much for this PR.